### PR TITLE
minor grammar changes

### PR DESCRIPTION
**Pull request by arronm**
Revised directions to read more clearly.

### CODE REVIEW

File: development/cpp/configuring_an_ide.rst [hunk 1]
```diff
@@ -83,13 +83,13 @@ from the upstream git repository or a specific fork etc. However this
 brings a little problem with it: as the development continues, source files
 (and folders) are added or removed. These changes needs to be reflected in
 your project files for Qt Creator too, so you continue to have a nice
-experience coding in it. A simple way to check these things, is to right click
+experience coding in it. A simple way to check, is to right click
```
Comment (Calinou) [inline development/cpp/configuring_an_ide.rst]: The comma isn't necessary here:

```suggestion
experience coding in it. A simple way to check is to right click
```

File: development/cpp/configuring_an_ide.rst [hunk 2]
```diff
@@ -270,29 +270,29 @@ Visual Studio Code
...
 
 - Update *launch.json* to match:
 
 .. image:: img/vscode_2_launch.json.png
 
-(Note that *godot.x11.tools.64* in "program" value might be differently called on macOS or Windows)
+(Note that *godot.x11.tools.64* in "program" value might be named differently on macOS or Windows)
 
-- Create *tasks.json* by starting Debug process (F5). VS Code will show dialog with *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
+- Create *tasks.json* by starting the Debug process (F5). VSCode will show a dialog with a *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
```
Comment (Calinou) [inline development/cpp/configuring_an_ide.rst]: "VS Code" is generally preferred to "VSCode" as a short form for "Visual Studio Code":

```suggestion
- Create *tasks.json* by starting the Debug process (F5). VS Code will show a dialog with a *Configure Task* button. Tap it and select *Create tasks.json file from template*, then select *Others*
```

**Comment by arronm** [comment]
👍

**Comment by Calinou** [comment]
Looks good to me, please squash the commits together so we can merge this. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions :slightly_smiling_face:

**Comment by arronm** [comment]
@Calinou - Thanks! I believe it should be squashed correctly now.

**Comment by Calinou** [comment]
Thanks!

**Pull request merged**